content/en/docs/reference/networking/service-protocols.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Protocols for Services
+content_type: reference
+---
+
+<!-- overview -->
+If you configure a {{< glossary_tooltip text="Service" term_id="service" >}},
+you can select from any network protocol that Kubernetes supports.
+
+Kubernetes supports the following protocols with Services:
+
+- [`SCTP`](#protocol-sctp)
+- [`TCP`](#protocol-tcp) _(the default)_
+- [`UDP`](#protocol-udp)
+
+When you define a Service, you can also specify the
+[application protocol](/docs/concepts/services-networking/service/#application-protocol)
+that it uses.
+
+This document details some special cases, all of them typically using TCP
+as a transport protocol:
+
+- [HTTP](#protocol-http-special) and [HTTPS](#protocol-http-special)
+- [PROXY protocol](#protocol-proxy-special)
+- [TLS](#protocol-tls-special) termination at the load balancer
+
+<!-- body -->
+## Supported protocols {#protocol-support}
+
+There are 3 valid values for the `protocol` of a port for a Service:
+
+### `SCTP` {#protocol-sctp}
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+When using a network plugin that supports SCTP traffic, you can use SCTP for
+most Services. For `type: LoadBalancer` Services, SCTP support depends on the cloud
+provider offering this facility. (Most do not).
```

Comment:
don't you think that is better if "you should check your cloud provider supports it` instead of asserting "most do not"?

Reply:
This wording is copied unchanged from [the original](https://kubernetes.io/docs/concepts/services-networking/service/#sctp).